### PR TITLE
TF Keras generic_utils_test missing test cases add

### PR DESCRIPTION
**Pull request by Dayananda-V**
test case added for
1-func_dump
2-func_load

### CODE REVIEW

**Comment by Dayananda-V** [comment]
@fchollet 

Please help to review the test case

File: tensorflow/python/keras/utils/generic_utils_test.py [hunk 1]
```diff
@@ -74,12 +73,116 @@ class CustomClass(object):
...
+    serialized = generic_utils.serialize_keras_object(None)
     self.assertEqual(serialized, None)
-    deserialized = keras.utils.generic_utils.deserialize_keras_object(
-        serialized)
+    deserialized = generic_utils.deserialize_keras_object(serialized)
     self.assertEqual(deserialized, None)
 
 
+class FuncDumpAndLoadTest(test.TestCase, parameterized.TestCase):
+  @parameterized.parameters(['simple_function', 'closured_function'])
```
Comment (fchollet) [inline tensorflow/python/keras/utils/generic_utils_test.py]: Add line break above

File: tensorflow/python/keras/utils/generic_utils_test.py [hunk 2]
```diff
@@ -74,12 +73,116 @@ class CustomClass(object):
...
+class ProgbarTest(test.TestCase):
+
+  def test_progbar(self):
+    values_s = [None,
+                [['key1', 1], ['key2', 1e-4]],
+                [['key3', 1], ['key2', 1e-4]]]
+
+    for target in (len(values_s) - 1, None):
+      for verbose in (0, 1, 2):
+        bar = generic_utils.Progbar(
```
Comment (fchollet) [inline tensorflow/python/keras/utils/generic_utils_test.py]: For a progbar test, you should catch the stdout and check that the display matches what you expect in the different verbosity modes.

File: tensorflow/python/keras/utils/generic_utils_test.py [hunk 3]
```diff
@@ -74,12 +73,116 @@ class CustomClass(object):
...
+        for current, values in enumerate(values_s):
+          bar.update(current, values=values)
+
+
+class MakeBatchesTest(test.TestCase):
+
+  def test_make_batches(self):
+    bathces = generic_utils.make_batches(100, 32)
+    self.assertEqual(len(bathces), 4)
+    self.assertEqual(bathces[0][1]-bathces[0][0], 32)
```
Comment (fchollet) [inline tensorflow/python/keras/utils/generic_utils_test.py]: - Fix typo
- For PEP8 style compliance, please insert spaces around operator `-`

File: tensorflow/python/keras/utils/generic_utils_test.py [hunk 4]
```diff
@@ -74,12 +73,116 @@ class CustomClass(object):
...
+
+    input_a = [None, [1, 2, 3], None, [1, 2]]
+    self.assertEqual(generic_utils.slice_arrays(input_a, 0),
+                     [None, [1, 2, 3], None, [1, 2]])
+    self.assertEqual(generic_utils.slice_arrays(input_a, 0, 1),
+                     [None, [1], None, [1]])
+    self.assertEqual(generic_utils.slice_arrays(input_a, stop=2),
+                     [None, [1, 2], None, [1, 2]])
+
+    with self.assertRaises(ValueError):
```
Comment (fchollet) [inline tensorflow/python/keras/utils/generic_utils_test.py]: When testing exceptions, match the error against a regex.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by hgadig** [comment]
@fchollet  Please review and approve this PR or suggest changes(if required)

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Review by pavithrasv — APPROVED** [review]

**Comment by rthadur** [comment]
@Dayananda-V can you please resolve conflicts ?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
